projects/ngneat/svg-icon/src/lib/svg-icon.component.ts

```diff
@@ -32,11 +27,9 @@ export class SvgIconComponent {
   @Input()
   set key(name: string) {
     if (this.registry.get(name)) {
```

Comment:
```ts
const icon = this.registry.get(name) ?? this.registry.get(this.config.missingIcon?.name)

if(icon) {
  this.element.setAttribute('aria-label', `${name}-icon`);
  this.element.classList.remove(`svg-icon-${this.lastKey}`);
  this.lastKey = name;
  this.element.classList.add(`svg-icon-${name}`);
  this.element.innerHTML = this.registry.get(name);
}
```

Reply:
This will fail because of registry.get method cannot handle with "null". But I can change registry.get method.